tests/test_autotuner.py

```diff
@@ -7,7 +7,7 @@
 from devito.core.autotuning import options  # noqa
 
 
-@switchconfig(log_level='DEBUG')
+@switchconfig(log_level='DEBUG', develop_mode=True)
```

Comment:
It needs to be documented that setting log_level='DEBUG' also turns on develop_mode.